chapters/VK_EXT_display_control/swapchain_counters.txt

```diff
@@ -43,7 +43,7 @@ include::{generated}/api/protos/vkGetSwapchainCounterEXT.txt[]
   * pname:pCounterValue will return the current value of the counter.
 
 If a counter is not available because the swapchain is out of date, the
-implementation may: return ename:VK_ERROR_OUT_OF_DATE_KHR.
+implementation must: return ename:VK_ERROR_OUT_OF_DATE_KHR.
```

Comment:
When we drafted the OUT_OF_DATE language, some IHVs were uncomfortable with requiring errors in states that are defined as OUT_OF_DATE, and carry on as if things are fine instead. This language is fine with me, but other driver vendors may still object to such requirements.

Reply:
OK. Good history. The issue by KrO0ze identifying non-exact language. Do we know who objected? see if they reaffirm stance or not.

Reply:
Lot of code relies on mustiness of this. So if you preserve "may", it needs some serious acompanying note what exactly that implies as well as what is the correct way to deal with swapchain state.

I can imagine though it could be allowed to be deferred. E.g. `acquire` pretends success, and then it fails at `present`. Still needs to be copiously noted in spec if that can happen.

Reply:
@cubanismo - lets see if we can craft some test that will work here.

Reply:
Not sure if resolved, but I certainly am happy with "must"...